### DISCUSSION TITLE
Not getting the connection lines

### DISCUSSION THREAD

**Discussion by gopusrikanth04** [xyflow/xyflow#1106] [Q&A]
Hello Team,

![react flow](https://user-images.githubusercontent.com/47174113/115104288-1d824a00-9f75-11eb-9bb3-0b4e9717c7ec.png)

 I have a requirement where it needs to be implemented like the image provide. The implemented code sample is here https://codesandbox.io/s/silent-mountain-67763 and also needed custom arrowheads like in image  

how to solve this problem
thanks

**Comment by moklick** [comment]
Hey. What is the question? You don't want to display the handles?

**Reply by gopusrikanth04** [reply]
Hello Moritz Klack,
 
  I have an requirement like 

![image](https://user-images.githubusercontent.com/47174113/115219137-d6d35200-a124-11eb-8b86-403008b5a17d.png)

https://codesandbox.io/s/silent-mountain-67763

But I am getting   

![Capture](https://user-images.githubusercontent.com/47174113/115221381-3894bb80-a127-11eb-96cd-92a9d3326386.PNG)


not getting connection lines to first line of blocks

How to solve this problem 

thanks

**Reply by moklick** [reply]
There are no connection lines because the custom node `CustomHeadNodeComponent` doesn't have handles. We need the handles for the position of the edges. You set the opacity to 0 if you don't want to display them but you need to add them as components to your custom node.

**Reply by gopusrikanth04** [reply]
thank you
